website/docs/r/event_trigger.html.markdown

```diff
@@ -114,8 +114,8 @@ resource "mongodbatlas_event_trigger" "test" {
 * `config_operation_types` - (Optional) Required for `DATABASE` type. The [database event operation types](https://docs.mongodb.com/realm/triggers/database-triggers/#std-label-database-events) to listen for. This must contain at least one value. Possible Values: `INSERT`, `UPDATE`, `REPLACE`, `DELETE`
 * `config_operation_type` - (Optional) Required for `AUTHENTICATION` type. The [authentication operation type](https://docs.mongodb.com/realm/triggers/authentication-triggers/#std-label-authentication-event-operation-types) to listen for. Possible Values: `LOGIN`, `CREATE`, `DELETE`
 * `config_providers` - (Optional) Required for `AUTHENTICATION` type. A list of one or more [authentication provider](https://docs.mongodb.com/realm/authentication/providers/) id values. The trigger will only listen for authentication events produced by these providers.
-* `config_database` - (Optional) Required for `DATABASE` type. The name of the MongoDB database that contains the watched collection.
-* `config_collection` - (Optional) Required for `DATABASE` type. The name of the MongoDB collection that the trigger watches for change events. The collection must be part of the specified database.
+* `config_database` - (Optional) Required for `DATABASE` type. The name of the watched MongoDB database.
```

Comment:
i think it's confusing (Optional) Required for....     together.   I'd remove all (Optional), i think it can be understood that it's optional for the other cases

Reply:
makes sense, will adjust